Source/JavaScriptCore/assembler/ARM64Assembler.h

```diff
@@ -41,9 +42,9 @@
 #include <zircon/syscalls.h>
 #endif
 
-#define CHECK_DATASIZE_OF(datasize) static_assert(datasize == 32 || datasize == 64)
+#define CHECK_DATASIZE_OF(datasize) static_assert(datasize == 32 || datasize == 64 || datasize == 128)
```

Comment:
This liberalize this macro too much because some instructions that do checks with this macro don't support a 128 data size.  Instead, let's introduce a different macro that uses CHECK_DATASIZE_OF and adds the 128 case.  Call it CHECK_VECTOR_DATASIZE_OF?